semantic-source/src/Source/Source.hs

```diff
@@ -76,7 +75,7 @@ totalRange = Range 0 . B.length . bytes
 
 -- | Return a 'Span' that covers the entire text.
 totalSpan :: Source -> Span
-totalSpan source = Span (Pos 1 1) (Pos (Prelude.length ranges) (succ (end lastRange - start lastRange))) where
```

Comment:
Oops, these changes are carted over from some earlier efforts. I'm going to revert any changes within `semantic-source`.